docs/Containers/WireGuard.md

```diff
@@ -540,12 +528,12 @@ $ docker exec wireguard bash -c 'apt update ; apt install -y tcpdump'
 To monitor traffic:
 
 ```bash
```

Comment:
Could you change this to `console` instead of `bash`. I'm changing the rest in #547. Can't change this one, as it's on an adjacent line to your change. Overlapping or adjacent changes will cause merge-conflicts.